docs/ru/engines/table-engines/mergetree-family/graphitemergetree.md

```diff
@@ -128,18 +131,62 @@ default
 При обработке строки ClickHouse проверяет правила в разделе `pattern`. Каждый `pattern` (включая `default`) может содержать параметр агрегации `function`, параметр `retention`, или оба параметра одновременно. Если имя метрики соответствует шаблону `regexp`, то применяются правила `pattern`, в противном случае правило `default`.
 
 Поля для разделов `pattern` и `default`:
-
+-   `rule_type` - тип правила (применяется только к метрикам указанных типов), используется для разделения правил проверки плоских/теггированных метрик.<br />
+                  Опциональное поле (для default не задается или задается тип `all`). Если используются метрики только одного типа или производительность проверки правил некритична, можно не использовать.<br />
+                  Если не для одного поля не задан тип правила или задаан тип `all`, создается одна цепочка правил для проверки (общая).<br />
+                  Иначе создаются 2 цепочки типов правил - для плоских и теггированных метрик (правило по умолчанию или правила с типом `all` покадают в обе цепочки).<br />
+                  Возможные значения<br />
+          `all` - универсальное правило, назначается также по умолчанию, если поле не задано<br />
+          `plain` - правило для плоских метрик (без тегов)<br />
+          `tagged` - правило для теггированных метрик (в формате `someName?tag1=value1&tag2=value2`)<br />
+          `tag_list` - правило для теггированных метрик (простой DSL для упрощения задания регулярного выражения), при старте транслируется в правило `tagged`
```

Comment:
Here are some formatting hints:

- `<br />` can be replaced with double trailing space
- values description can be shifted left, so it's visible

I hope, you find it a bit more clear:

```suggestion
- `rule_type` - тип правила (применяется только к метрикам указанных типов), используется для разделения правил проверки плоских/теггированных метрик. Опциональное поле. Значение по умолчанию: `all`.
Если используются метрики только одного типа или производительность проверки правил некритична, можно не использовать. По умолчанию создается только один тип правил для проверки. Иначе, если хотя бы для одного правила указано отличное от умолчания значение, создаются 2 независимых типа правил - для обычных (классические root.branch.leaf) и теггированных метрик (root.branch.leaf;tag1=value1).  
Правила по умолчанию попадают в оба правила обоих типов.  
Возможные значения:
    - `all` (default) - универсальное правило, назначается также по умолчанию, если поле не задано
    - `plain` - правило для плоских метрик (без тегов), regexp
    - `tagged` - правило для теггированных метрик (в формате `someName[?]tag1=value1&tag2=value2`), regexp
    - `tag_list` - правило для теггированных метрик (в формате тегов graphite `someName;tag1=value1;tag2=value2`), простой DSL для упрощения задания регулярного выражения. При процессинге транслируется в правило `tagged`, 
```